### PR TITLE
Convert image

### PR DESCRIPTION
**Pull request by ChristianTC**
Could you review it, please.

### CODE REVIEW

File: CONVERT_SERVICE/convert_service/convert_app/model/convert_image/convert_image.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 # accordance with the terms of the license agreement you entered into
 # with Jalasoft.
 #
+import datetime
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_app/model/convert_image/convert_image.py]: please delete this empty line. And change the filename in the copyright.

File: CONVERT_SERVICE/convert_service/convert_app/model/convert_image/convert_image.py [hunk 2]
```diff
@@ -72,70 +73,97 @@ def get_vignette(self):
         return self.vignette
 
 
+"""Class to convert image with many params"""
 class ConvertImage:
```
Comment (lcarrietas) [inline CONVERT_SERVICE/convert_service/convert_app/model/convert_image/convert_image.py]: Please let's use only one class per module.

**Review by lcarrietas — COMMENTED** [review]
Please delete CONVERT_SERVICE/convert_service/convert_app/model/convert_image/convert_image.py
and check the other comments.

File: CONVERT_SERVICE/convert_service/convert_app/views.py [hunk 3]
```diff
@@ -10,12 +10,17 @@
...
 
+from urllib import request
+from django.contrib.sites.shortcuts import get_current_site
 from django.core.files.storage import FileSystemStorage
 from pathlib import Path
 from django.views import View
 from django.http import HttpResponse
 from .model.ffmpeg.ffmpeg_execute import ffmpegexecute
 from .model.ffmpeg.img_compresor import zip_dir
+
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_app/views.py]: Please remove this extra line.

File: CONVERT_SERVICE/convert_service/convert_app/views.py [hunk 4]
```diff
@@ -10,12 +10,17 @@
...
+from .model.convert_image.convert_image import Param, ConvertImage
+
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_app/views.py]: Please remove this extra line.

File: CONVERT_SERVICE/convert_service/convert_service/urls.py [hunk 5]
```diff
@@ -27,8 +27,10 @@
...
 from django.contrib import admin
 from django.urls import path, include
+from django.conf.urls.static import static
+from django.conf import settings
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('ffmpeg/', include('convert_app.urls')),
-]
+                  path('admin/', admin.site.urls),
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_service/urls.py]: Please remove the extra indentation

**Review by Franco98404 — COMMENTED** [review]
Please add this file to gitignore, or is necessary?
CONVERT_SERVICE/convert_service/convert_app/model/convert_image/minions.png

File: CONVERT_SERVICE/convert_service/convert_app/views.py (hunk 4 above)
Comment (Franco98404) [reply CONVERT_SERVICE/convert_service/convert_app/views.py]: Please remove this extra line @ChristianTC

**Review by lcarrietas — APPROVED** [review]
Good!

**Review by Franco98404 — APPROVED** [review]
Looks good! 👍

**Pull request merged**